src/dotnet-toolbox/VersionFile/VersionFile.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace DotNetToolbox.VersionMetadata
+{
+    public class VersionFile
+    {
+        private static List<PackageMetadata> _contents = new List<PackageMetadata>();
```

Comment:
Is there a reason to have this static instance? JSON.NET will create a new instance regardless. Suggest just making this a local variable in the WriteVersion method.